program/src/state.rs

```diff
@@ -227,12 +223,19 @@ pub struct HolderRewards {
     /// Stored as a `u128`, which includes a scaling factor of `1e9` to
     /// represent the exchange rate with 9 decimal places of precision.
     pub last_rewards_per_token: u128,
-    /// The last seen total rewards amount in the aggregate holder rewards
-    /// account.
-    pub last_seen_total_rewards: u64,
     /// The amount of unharvested rewards currently stored in the holder
     /// rewards account that can be harvested by the holder.
     pub unharvested_rewards: u64,
+    _padding: u64,
```

Comment:
So, this is a bit annoying -- the VM doesn't support 16-byte aligned structs so this will actually align to 8 in BPF, but 16 on other architectures. It was 16 on my x86 machine, and 8 when I printed from inside the VM.

So this doesn't need padding, but it's probably not a huge deal

Reply:
Unfortunately bytemuck's derive macros panic on this if we don't add the padding.

```
cannot transmute between types of different sizes, or dependently-sized types
source type: `HolderRewards` (256 bits)
target type: `state::_::{closure#0}::TypeWithoutPadding` (192 bits)
```

Is there a workaround I'm missing for this?

Reply:
Ah interesting, I wasn't aware of that! All good then, let's keep it